### DISCUSSION TITLE
“双击全屏”相关

### DISCUSSION THREAD

**Discussion by WDonny** [the1812/Bilibili-Evolved#2459] [提问 / Q&A] [answered]
由于不太确定这是否为一个BUG所以先发在讨论区
开启“双击全屏”功能但不开启“双击时阻止单击事件”选项时，在播放器上使用鼠标单击对播放状态进行改变（如暂停时播放或播放时暂停）时会在播放状态短暂切换后恢复到原有的播放状态（即原本暂停的视频点击后会短暂播放约半秒后回到暂停状态，原本播放中的视频同理）
开启“双击时阻止单击事件”选项后恢复正常，可以在播放器页面通过单击进行视频的播放和暂停，故请问该开关的描述/功能实现是否有误，还是我对该开关描述的理解存在问题？

**Comment by the1812** [comment, marked as answer]
功能实现有误
